packages/forms-web-app/src/views/components/ui/step-by-step-navigation.njk

```diff
@@ -1,11 +1,15 @@
-{% macro stepByStepNavigation(params, activeStep, template, state) %}
+{% macro stepByStepNavigation(params, activeStep, template, state, t) %}
   {% set navigationSize = "" %}
 
   {% if template === "main" %}
     {% set navigationSize = "app-step-nav--large" %}
   {% endif %}
 
   {% set navigationState = "" %}
+  {% set showText = t('common.showButton')%}
+  {% set hideText = t('common.hideButton')%}
+  {% set showAllText = t('common.showAllButton')%}
+  {% set hideAllText = t('common.hideAllButton')%}
```

Comment:
These only look like they are used once on the template? Do we need to create variables for them?